src/app/pages/credentials/backup-credentials/cloud-credentials-form/oauth-provider/oauth-provider.component.ts

```diff
@@ -32,6 +35,21 @@ export class OauthProviderComponent {
 
   readonly helptext = helptext;
 
+  get isGooglePhotosProvider(): boolean {
```

Comment:
No point in changing this component since it's very different from the original implementation. You need to create a new component.